src/banner.js

```diff
@@ -1,19 +1,30 @@
-const bannerTemplate = ` ___  _          _      _                                       
-|  _|| |        |_|    | | ###################################  
-| |_ | |  _   _  _  ___| |     _____  _   _  ____  _   _  ____  
-|  _|| | | | | || |/  _  | ___|  _  || | | ||  _ || | | ||  _ | 
-| |  | |_| |_| || || |_| |/__/| |_| || |_| ||  __/| |_| ||  __/ 
-|_|  |__/|_____/|_/|_____/    \\___  ||_____/|____||_____/|____| 
-                                  | |                           
```

Comment:
The trailing space was on purpose to make each line exactly the same length, but i guess it shouldn't really matter.
I was thinking maybe if the terminal is smaller and the line breaks that the breaking pattern would be the same for each line at least, but at that point it might not be readable anyways.
I haven't really tested this.